server/api/controllers/member.js/add-member.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  friendlyName: 'Add members',
+  description: '',
+  inputs: {
+    member: {
+      description: '',
+      type: [{
+          fullName: 'string',
+          email: 'string',
+          campus: 'string',
+          lifestage: 'string',
+          isMember: 'boolean',
+          isBaptised: 'boolean',
+          phoneNumber: 'number',
+          //classesTaken: 'json'
+      }]
+    }
+  },
+  exits: {},
+  fn: async function(inputs, exits) {
```

Comment:
can use `{}` instead to destructure objects. e.g: `async function({ member }, exits) {`